benchmark/benchmarks.dart

```diff
@@ -38,7 +38,7 @@ class InsertBenchmark extends RTreeBenchmarkBase {
       int width = rand.nextInt(100);
       RTreeDatum item =
           RTreeDatum<String>(Rectangle(x, y, width, height), 'item $i');
-      tree.insert(item);
+      tree.insert(item as RTreeDatum<String>);
```

Comment:
You don't need to cast this if you remove the type declaration on line 39. I'd go ahead and use `final` as well.